### ISSUE TITLE
Saving Structured Grading Instructions does not work properly

### ISSUE THREAD

**Issue by FlorianGlombik** [ls1intum/Artemis#3291]
### Describe the bug

Sometimes changes to _Structured Grading Instructions_ are not saved.

#### To Reproduce

1. Create an exercise with Structured Grading Criteria (e.g. modeling / text exercise)
2. Activate _Grading Instructions_
3. Modify the grading instructions and save the changes (and don't preview the changes)
4. The changes were not saved properly

It seems like the last entered element (letter / copy pasted element) is not saved, except the preview is displayed.
#### Expected behavior

The _Grading Instructions_ should be saved properly.

##### Last letter not saved
The last entered letter was not saved. When just adjusting the score of an instruction, this leads to the score being 0 since the last letter was not saved properly.
![StructuredGradingCriteria-lastLetterNotSaved](https://user-images.githubusercontent.com/63976129/115993866-fc54d580-a5d4-11eb-8b49-7fe8404b0db6.gif)

##### Last copied element not saved
![StructuredGradingCriteria-SavingCopyPastedInstructionDidNotWork](https://user-images.githubusercontent.com/63976129/115993927-3a51f980-a5d5-11eb-9da5-22ce67aea5d8.gif)

The element is only saved if it is edited, but then the last entered letter is not saved.
![StructuredGradingCriteria-SavingNewInstructionWorked](https://user-images.githubusercontent.com/63976129/115993956-56ee3180-a5d5-11eb-8d3b-df6b8761b4d0.gif)

##### Saving works when preview is displayed
![StructuredGradingCriteria-SavingCopyPastedInstructionWorksWithPreview](https://user-images.githubusercontent.com/63976129/115993973-68373e00-a5d5-11eb-9be9-61a6d15340a5.gif)

### Environment

<details><pre>
 - OS: Windows 10
 - Browser Chrome, latest
 - ArtemisVersion 4.11.8
</pre></details>

**Comment by krusche** [comment]
might be a change tracking issue in Angular. Did you change the focus? (this typically triggers the change from the UI element into the client model

**Comment by krusche** [comment]
@imstreetspirit can you please check if this issue still appears after merging #3294?
If yes, it would be nice, if we could fix it.

**Comment by imstreetspirit** [comment]
> @imstreetspirit can you please check if this issue still appears after merging #3294?
> If yes, it would be nice, if we could fix it.

While I was working on  #3294, I've encountered with this issue. I found that the last character is updated after the markdown change sent the event. There was a changes for markdown editor when the strict templates changes happened. I changed the order of two lines, and I believe it solves the problem, cause the reviewers did not mentioned them after I fixed it.
